src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -362,27 +359,10 @@ private void processReset(
         }
     }
 
-    // Map http1.1 header to http2 header field in HEADERS request
-    private void mapHeader(
-        HpackContext hpackContext,
-        HttpHeaderFW httpHeader)
-    {
-        http2HeadersRW.header(mapConsumer(hpackContext, httpHeader));
-    }
-
-    // Map http1.1 header to http2 header field in PUSH_PROMISE request
-    private void mapPushHeader(
-        HpackContext hpackContext,
-        HttpHeaderFW httpHeader)
-    {
-        pushPromiseRW.header(mapConsumer(hpackContext, httpHeader));
-    }
-
-    private Consumer<HpackHeaderFieldFW.Builder> mapConsumer(
-        HpackContext hpackContext,
-        HttpHeaderFW httpHeader)
+    // Map http1.1 header to http2 header field in HEADERS, PUSH_PROMISE request
+    private BiFunction<HttpHeaderFW, HpackHeaderFieldFW.Builder, HpackHeaderFieldFW> mapHeader(HpackContext hpackContext)
     {
-        return builder ->
+        return (httpHeader, builder) ->
         {
```

Comment:
Move this implementation to a separate instance method, use implicit `hpackContext` which is already in scope via instance field (no need to pass as parameter).
Then pass instance method reference directly from `.set(...)` at line 284 instead of wrapping with `mapHeader(...)`.